includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -1332,9 +1332,11 @@ protected function build_display_items( $itemized_display_items = false ) {
 
 				$product_name = $cart_item['data']->get_name();
 
+				$item_tax = wc_tax_enabled() && 'incl' !== get_option( 'woocommerce_tax_display_cart' ) ? 0 : $cart_item['line_subtotal_tax'];
```

Comment:
Maybe you can move this piece of code: `wc_tax_enabled() && 'incl' !== get_option( 'woocommerce_tax_display_cart' )` into a separate function to prevent code duplication (check line 1362) which will make refactoring (if needed) in future easier.

It would also be nice to add check if `$cart_item['line_subtotal_tax']` exists before or write something like this

`$item_tax = wc_tax_enabled() && 'incl' !== get_option( 'woocommerce_tax_display_cart' ) ? 0 : ($cart_item['line_subtotal_tax']  ?? 0 ); `

Reply:
Done

Reply:
Done.